elements/box/README.md

```diff
@@ -0,0 +1,37 @@
+The `box` element can contain all the another elements except the `full width` element. The primary function of the `box` element is wrapping its children elements with a HTML table structure because some of the CSS rules only apply on `<table>` or `<td>` elements (mostly on Outlooks).
+
+This wrapper table helps:
+- positioning its children elements inside the email layout
+- apply padding, margin, border and background setting around different element
+- added class attributes take place on the wrapper table on the proper level (`<table>` or `<td>`)
+
+
+Example:
+```
+{
+"type":"BOX",
+"children":[
+    ...
+],
+"background":{
+    ...
+},
+"border":{
+    ...
+},
+"margin":{
+    ...
+},
+"padding":{
+    ...
+},
+"hideOnDesktop":false,
+"hideOnMobile":false
+}
+```
+| Properties    | Type          | Values | Description
+| ------------- |:-------------:| -----:|
+| background | object | object | [Read more](../prop-groups/background.md)
```

Comment:
I would always go from the outside to the inside of the element when it comes to talk about the properties...
margin -> border -> padding -> background -> children

If we follow this simple rule, then it will always be the same order in the docs when you talk about it. I suggest you to follow a similar order then you show example jsons. The extra props can go even before the margin. (type, hideOnDesktop, hideOnMobile.)